PagerDuty/public-support-scripts
# Add execute permissions to non-executable scripts

**Link back to Jira ticket:** N/A

## Description

While testing a few scripts, I noticed that several of them didn't have execute permissions on them, which prevented them from being run via the command line as a standalone program (without the interpreter), which is mentioned as a given here: https://github.com/PagerDuty/public-support-scripts#howto

## Implementation

There were five scripts that needed `chmod +x`, nothing else changes here.

### Steps to test changes
* Pull down and run the scripts changed here from master vs this branch. You will need to use the interpreter on master, this branch fixes that.
* As a note, the diff looks weird (Empty file??) when changing only permissions. This seems to be a known issue with GitHub: https://github.com/github/feedback/discussions/6814... when you click on "view file" in the three-dot menu on the right-hand side of the diff, you see the whole contents still.

## Documentation 
- [x] There is NO documentation that needs to be updated upon merging these changes (this ensures compliance with our current docs in this repo)
